src/components/MovieForm.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { FormEvent } from 'react';
+import '../components-styles/MovieForm.scss';
+import {GenreList} from '../constants'
```

Comment:
For const we should use upper case
```suggestion
import {GENRE_LIST} from '../constants'
```